tests/integration/pkg/helm_1_test.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+)
+
+const (
+	yamlPathReceiverMock = "yamls/receiver-mock.yaml"
```

Comment:
Is this currently used for anything?

Reply:
This can be treated as WIP :)

Reply:
You want to add anything using it to this PR or merge it as-is?

Reply:
I've removed this (I have this being used in a separate test that I've prepared).